tests/waku_filter_v2/test_waku_client.nim

```diff
@@ -2076,10 +2076,10 @@ suite "Waku Filter - End to End":
             contentTopic = contentTopic, payload = getByteSequence(100 * 1024)
           ) # 100KiB
           msg4 = fakeWakuMessage(
-            contentTopic = contentTopic, payload = getByteSequence(MaxRpcSize - 1024)
+            contentTopic = contentTopic, payload = getByteSequence(MaxPushSize - 1024)
```

Comment:
May elaborate on this plz?

Reply:
`MaxPushSize` has replaced `MaxRpcSize` for the new filter protocol (because we don't have an RPC anymore, but rather two different protocols for the subscribe request-response and the actual message pushes). It was a mistake to use `MaxRpcSize` in this file from the beginning as it has nothing to do with the new filter protocol. Why this max is needed in the first place is a bit out of scope of this PR, but this relates to how libp2p decodes byte streams.